apps/expo/src/app/index.tsx

```diff
@@ -54,7 +59,9 @@ export default function Page(): ReactNode {
 								scale: { type: "spring", delay: 350 },
 							}}
 						>
-							<Text className="text-8xl font-bold text-violet-700">
+							<Text
+								className={styles.logo + " " + "text-primary"}
```

Comment:
try to apply the way too avoid adding  in the class name, just mane a new option, it's calm